src/screens/FeedScreen/hooks/useCalculationContent.js

```diff
@@ -1,4 +1,5 @@
 import React from 'react';
+
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The import statement for `React` is unnecessary since you're not using JSX or any React features in this file.
```diff
- import React from 'react';
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->